README.md

```diff
@@ -1,82 +1,209 @@
-# Variant API Client
+# VarSome API Client
 
-## A basic api client implementation for [api.varsome.com](https://api.varsome.com)
+## A basic API client implementation for [api.varsome.com](https://api.varsome.com)
 
-This client is still in beta but it is a good start to start playing around with the API.
+This client is still in beta but it is a good for playing around with the API.
+
+### Python versions
+Python version 3 is supported.
```

Comment:
Requires at least Python 3.5, you can download the latest version from www.python.org

Reply:
ie: Python 3.4 doesn't work.